sdk/python/cmd/pulumi-language-python/main.go

```diff
@@ -1009,3 +1017,154 @@ func (host *pythonLanguageHost) RunPlugin(
 
 	return closer.Close()
 }
+
+func (host *pythonLanguageHost) GenerateProject(
+	ctx context.Context, req *pulumirpc.GenerateProjectRequest,
+) (*pulumirpc.GenerateProjectResponse, error) {
+	loader, err := schema.NewLoaderClient(req.LoaderTarget)
+	if err != nil {
+		return nil, err
+	}
+
+	extraOptions := make([]pcl.BindOption, 0)
```

Comment:
optional: For `make([]T, 0)` , `var` form is better because it won't allocate unless appended to.

```go
var extraOptions []pcl.BindOption
```